index.html

```diff
@@ -59,8 +59,37 @@
     </a>
 
     <!-- Hero -->
-    <section id="hero" class="section-pad-left">
-
+    <section id="hero">
+        <div id="hero-content">
+            <h1>BrickHack 8</h1>
+            <h3>March 5 - 6, 2022 | RIT Online Hackathon</h3>
```

Comment:
Teensy nit, looks like there's a lot of visual space between ` 5 - 6, ` due to font; maybe no spaces around the dash?
<img width="331" alt="CleanShot 2021-11-29 at 01 30 29@2x" src="https://user-images.githubusercontent.com/2782749/143819360-db953d6b-7af8-4c04-ab44-6f8f25019d3b.png">

Reply:
That would be my personal writing style. Changing it